aspnetcore/tutorials/first-mvc-app/details.md

```diff
@@ -68,7 +68,7 @@ public async Task<IActionResult> Delete(int id, bool notUsed)
 
 ### Publish to Azure
 
-See [Publish an ASP.NET Core web app to Azure App Service using Visual Studio](xref:tutorials/publish-to-azure-webapp-using-vs) for instructions on how to publish this app to Azure using Visual Studio.  The app can also be published from the [command line](xref:tutorials/publish-to-azure-webapp-using-cli).
+For information on deploying to Azure, See [Create an ASP.NET Core web app in Azure](/azure/app-service/app-service-web-get-started-dotnet). You don't need to create a new app, start at [Launch the publish wizard](/azure/app-service/app-service-web-get-started-dotnet#launch-the-publish-wizard).
```

Comment:
That doc won't help you publish this app because this app has a SQL database.  There's a tutorial that does deploy an ASP.NET Core doc with SQL database -- https://docs.microsoft.com/en-us/azure/app-service/app-service-web-tutorial-dotnetcore-sqldb  -- but it uses CLI not VS.  I'm thinking the best thing here is to link to both, explaining the limitations of each.  ASP.NET Core with SQL to App Service is arguably a top task but we don't have anything for it. cc @cephalin any other ideas?

Reply:
Another idea is to point to https://docs.microsoft.com/en-us/azure/app-service/app-service-web-tutorial-dotnet-sqldatabase and explain that even though it shows a .NET Framework app the steps are exactly the same.

Reply:
For now I like the idea of linking to the .NET Framework doc; I think we can live with that risk and just change or remove the link if the tutorial is changed radically.